### ISSUE TITLE
Splash screen stays open indefinitely

### ISSUE THREAD

**Issue by i00** [project-sbc/Handheld-Control-Panel#33]
I have this installed on 3 computers, on 1 the splash screen never closes; if I press Alt+F4 is disappears, and everything still works!
Also can an option be added to disable this completely?

Thanks

**Comment by trias702** [comment]
Can confirm, I have the same issue.

Is this project still alive? There have been no updates since May 2023.

**Comment by NightHammer1000** [comment]
Yeah would like to know that too. Its an awesome project. Would hate to see it dead in the water.

**Comment by project-sbc** [comment]
Nah it’s been moved to another foundation. Not dead in the water as I’ve been putting in hours to rework it

Best Regards,
Chris
________________________________

**Comment by trias702** [comment]
@project-sbc Can you please provide the link to this other "foundation"? Would very much like to download an up-to-date version of this tool.
